src/gpu/intel/jit/codegen/codegen_extensions.hpp

```diff
@@ -13,24 +13,35 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *******************************************************************************/
+#ifndef GPU_INTEL_JIT_CODEGEN_CODEGEN_EXTENSIONS_HPP
+#define GPU_INTEL_JIT_CODEGEN_CODEGEN_EXTENSIONS_HPP
 
-#ifndef GPU_INTEL_REORDER_JIT_NORMALIZATION_HPP
-#define GPU_INTEL_REORDER_JIT_NORMALIZATION_HPP
-
-#include "gpu/intel/jit/ir/tensor.hpp"
+#include "gpu/intel/jit/codegen/operand.hpp"
+#include "gpu/intel/jit/codegen/register_scope.hpp"
+#include "gpu/intel/jit/ir/include/kernel.hpp"
+#include "oneapi/dnnl/dnnl_config.h"
 
 namespace dnnl {
 namespace impl {
 namespace gpu {
 namespace intel {
-namespace reorder {
 namespace jit {
 
-using namespace intel::jit;
-void normalize(layout_t &a, layout_t &b);
+struct codegen_extension_interface_t {
```

Comment:
Nit: `interface` -> `iface` could be more aligned with `expr_iface_t`, `kernel::iface_t`.